src/did/src/notify.rs

```diff
@@ -0,0 +1,118 @@
+use candid::{CandidType, Principal};
+use ethers_core::abi::ethabi::Bytes;
+use ethers_core::abi::{Error, Function, Param, ParamType, StateMutability, Token};
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+
+use crate::H256;
+
+/// Function signature, used if transaction should notify
+/// some principal with some transaction receipt.
+#[allow(deprecated)] // need to initialize `constant` field
+pub static NOTIFICATION: Lazy<Function> = Lazy::new(|| Function {
+    name: "evm_canister_notification_needed".into(),
```

Comment:
[minor] Should we use solidity name notation for it?

Reply:
We'll probably need to rename this NotificationInput